test/integ/pandas/test_arrow_pandas.py

```diff
@@ -1084,3 +1038,16 @@ def test_batch_to_pandas_arrow(conn_cnx, result_format):
                 assert arrow_table.shape == (10, 2)
                 assert arrow_table.column_names == ["FOO", "BAR"]
                 assert arrow_table.to_pydict()["FOO"] == list(range(rowcount))
+
+
+def test_simple_arrow_fetchall(conn_cnx):
+    rowcount = 100000
+    with conn_cnx() as cnx:
+        with cnx.cursor() as cur:
+            cur.execute(SQL_ENABLE_ARROW)
+            cur.execute(
+                f"select seq4() as foo from table(generator(rowcount=>{rowcount})) order by foo asc"
+            )
+            arrow_table = cur.fetch_arrow_all()
+            assert arrow_table.shape == (rowcount, 1)
+            assert arrow_table.to_pydict()["FOO"] == list(range(rowcount))
```

Comment:
I have added a simple test like this for now. The reason why is because `fetch_arrow_all` is a new cursor function and so I thought it would be valuable to add a basic test. In theory though the new `fetch_arrow_*` functions behave quite similarly to their `fetch_pandas_*` counterparts + go through the same code path as when you call `to_pandas/to_arrow` which is also tested, so perhaps this test is not so necessary.

Please let me know if you think there is any value to this test/more should be added since `fetch_arrow_*` are new/is trivial and not needed.